sysbuild/sdp.cmake

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+# If it is enabled, include the eGPIO FLPR application in the build
+if(SB_CONFIG_EGPIO_FLPR_APPLICATION)
+  # Extract soc name from related variables
+  string(REPLACE "/" ";" split_board_qualifiers "${BOARD_QUALIFIERS}") # Replace all "/" with ";"
+  list(GET split_board_qualifiers 1 target_soc) # Get the second element of the list, which is the soc name
```

Comment:
these comments are not helpful, comments should explain the reason or purpose for something, not say exactly what a line does which is already known by looking at what the line does